source/settings/core.rst

```diff
@@ -1483,6 +1479,25 @@ See :ref:`settings` for list of all setting groups.
        language_filters_en = lowercase snowball english-possessive stopwords
      }
 
+
+.. dovecot_core:setting:: language_filter_stopwords_dir
+   :seealso: @fts_tokenization
+   :values: @string
+
+     Path to the directory containing stopword files. Stopword files are
+     looked up in ``”<path>”/stopwords_<lang>.txt``.
```

Comment:
the quotes around `<path>` seem unnecessary. Although maybe the whole `<path>` is unnecessary and it should just be:

```
     Path to the directory containing stopword files. The files are named `stopwords_<lang>.txt`.
```